test/built-ins/RegExp/property-escapes/generated/strings/Basic_Emoji-negative-CharacterClass.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Mathias Bynens. All rights reserved.
```

Comment:
Question about the location of these new tests: should they go under [Test262’s `test/built-ins/RegExp/property-escapes/` directory](https://github.com/tc39/test262/tree/main/test/built-ins/RegExp/property-escapes) under a new `strings` subdirectory (like in the current patch), or in a new sibling directory alongside `property-escapes`, or somewhere else?

Reply:
For now the directory structure is fine. I doubt it will need to change, but if it does that's not a problem.